collections/build-collections.py

```diff
@@ -5,7 +5,7 @@
 
 
 def write_file(dict, path):
-    manifest_bucket = "mellon-manifest-prod-manifestbucket-1aed76g9eb0if"
+    manifest_bucket = "marble-manifest-prod-manifestbucket-lpnnaj4jaxl5"
```

Comment:
How durable is that bucket suffix? I'm wondering if this is something to consider for parameterization.